db/db.go

```diff
@@ -88,9 +88,9 @@ func (r Rep) ColumnString(col string) string {
 	case WPM:
```

Comment:
Printing out a pretty version of the contents of the rep should not be the responsibility of the database. What should happen is
- The rep contains the raw data from the database
- If you need to display the rep's data in a specific format, then pass that data into a function which displays it consistently for you based on the type of thing you're showing (wpm, accuracy, and so on)
`FormatAccuracy(float64 acc)` returns a string of `xx.xx%` for instance

This is because I'm doing calculations on ranges of wpm and so on from many rows. I want the output of an average of wpms to be the same as a single wpm.